packages/kbn-openapi-generator/src/cli.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import yargs from 'yargs/yargs';
+import { generate } from './openapi_generator';
+import { AVAILABLE_TEMPLATES } from './template_service/template_service';
+
+export function runCli() {
+  yargs(process.argv.slice(2))
+    .command(
+      '*',
+      'Generate code artifacts from OpenAPI specifications',
+      (y) =>
+        y
+          .option('rootDir', {
+            describe: 'Root directory to search for OpenAPI specs',
+            demandOption: true,
+            string: true,
+          })
+          .option('sourceGlob', {
+            describe: 'Elasticsearch target',
+            default: './**/*.schema.yaml',
```

Comment:
I didn't read close enough the first pass through and created an `assistant.schema.yml` and it wasn't getting picked up since it wasn't `.yaml`. After some searching it seems `yaml` is the defacto/preferred extension even though we have a lot of `.yml` files floating around the Kibana repo. nit here would be maybe also support `.yml`, or print an info/warning message around `🕵️‍♀️   Found 0 schemas, parsing`, but this is entirely user error, so feel free to do nothing here 🙂

Reply:
Adding `.yml` support should be pretty straightforward. I can tackle that in subsequent PRs, since work on the package is still ongoing.